R/chromote-fbref.R

```diff
@@ -0,0 +1,55 @@
+
+#' @source <https://github.com/tidyverse/rvest/blob/chromote/R/chromote.R>
+#' @importFrom rlang check_installed
+#' @noRd
+worldfootballr_chromote_session <- function(url) {
+  rlang::check_installed(c("chromote", "R6"))
```

Comment:
i don't want require these packages since we only use them for the new `fb_league_stats` function, so i added this function to prompt the user to install these additional packages if they don't have them already

Reply:
Neat